README.md

```diff
@@ -46,10 +46,10 @@ Mandatory flags:
 Optional flags:
 -r, --root          Local directory to store testnet data in (IMPORTANT: this directory is removed and recreated if --reset-chain is set)
 
---reset-chain       Delete local data to do a clean connect to the testnet
+--reset-chain       Delete local data to do a clean connect to the testnet (If you participated in an older version of the testnet)
```

Comment:
should add a warning on the flag and the script when it's used. 
it's a dangerous flag to use as it erases all secret keys also. 

i.e., when `reset-chain` flag is used, we pause, ask a question, `are you sure you want to erase all previous blockchain state?`, press Y to continue.